lib/components/atoms/Blockquote/tests/Blockquote.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Blockquote from '../Blockquote';
```

Comment:
The test cases should be running on Vanilla components.

Reply:
fixed